velodrome/token-counter/deployment.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         - --influx-password=$(INFLUXDB_ROOT_PWD)
         - --token=/github-tokens/k8s-merge-robot
```

Comment:
Also, I believe k8s-merge-robot is now k8s-github-robot, so the token for this one needs to be updated as well (maybe as a follow-up)?